pages/praharpatel.md

```diff
@@ -0,0 +1,11 @@
+# Prahar Patel
+
+Hello! I am from Los Angeles, CA. I am a junior at California State Polytechnic University, Pomona pursuing Bachelor's in Computer Science. 
+<li>   I am using Windows 10 with AMD A6-6310 APU and AMD Radeon R4 graphics. Clock speed @ 1.8 GHz and a 64 bit system. 
```

Comment:
@praharpatel markdown please

examples of other personal pages are in http://open-learning-exchange.github.io/#!pages/team.md